### ISSUE TITLE
TBTray does nothing? "v2"

### ISSUE THREAD

**Issue by ch3mn3y** [sagamusix/TBTray#5]
I seem to have similar issue to one reported by @mmodler.
Unfortunately :
1. moving TBtray out of TBs folder or even to the different nonUAC protected drive does nothing. There is no indication it is working (except TBtray.exe process in Task Manager).
2. mail.tabs.drawInTitlebar is set to false (I read the Readme file), but it didn't change a thing...
3. the windows ends in "- Mozilla Thinderbird".

Tried killing the task and restarting it. After about 10 moves of the files I tried registering it and rebooting OS. Still nothing.

I use x64 68.2.2 TB, meaning latest available stable version. It's cleanly installed to be sure there is no problems.

**Comment by sagamusix** [comment]
Are you using the TBTray version from the "64-bit" folder? If you do, I have no idea what else could cause the problem. If you have Visual Studio or any other debugger installed, you could double-check if TBTray.dll is successfully injected into Thunderbird by attaching the VS debugger to thunderbird.exe, pausing the execution and checking which modules are loaded - TBTray.dll should be among them.

**Comment by ch3mn3y** [comment]
I use x64 as this is the TB i have. Will check tomorrow (23/11PM here) with VS if it works.

**Comment by rokuoku** [comment]
Same issue here

**Comment by w4hnsinn** [comment]
so, on the off chance somebody else is like me, make sure that you use the TBTray version (32bit or 64bit) that your TB itself is.
I didn't even realize I was using the 32-bit TB still, because everything else I use is 64-bit .. D'OH!

**Comment by Marooned-MB** [comment]
I know you don't support this tool but following this thread I'll write that it also does not work for me.

I've tried both 32 and 64 bit version to be sure. `TBTray.exe` process is active, minimizing Thunderbird works as default, to task bar, no extra tray icon. Closing Thunderbird just closes it.

Setting `mail.tabs.drawInTitlebar` changes layout of Thunderbird by adding separate bar for title only (classic Windows look) but does not help TBTray to do its job.

I have English Windows but Polish locale so window title looks like this: `Opcje - Mozilla Thunderbid`.

However, [alternative solution](https://github.com/TheGuardianWolf/ThunderbirdTray) is working for me so I'm fine.
